src/lib/utils.ts

```diff
@@ -2,41 +2,68 @@ import lzString from 'lz-string';
 const { decompressFromEncodedURIComponent, compressToEncodedURIComponent } = lzString;
 
 const parseQuery = (
-	input: string
```

Comment:
Looks like the formatting has been changed here, probably just due to prettier not running. Once you're able to build the project properly, it should format through your IDE. Or you can run `yarn format` to fix them in one go.